pkg/operator/envtest/controller_test.go

```diff
@@ -132,41 +132,7 @@ var _ = Describe("Workload controller", func() {
 		Entry("Should create a config audit report Pod", "pod-configauditreport-expected.yaml"),
 		Entry("Should create a config audit report ReplicaSet", "replicaset-configauditreport-expected.yaml"),
 	)
-
-	NormalizeUntestableRbacAssessmentReportFields := func(ca *v1alpha1.RbacAssessmentReport) *v1alpha1.RbacAssessmentReport {
```

Comment:
@chen-keinan Since we have removed rbac scanner and rbac policies from defsec, I think it makes sense to remove the rbacAssessmentReport from Trivy-Operator too. Wdyt ?